src/panels/events.js

```diff
@@ -80,10 +73,11 @@ var EventsPanel = React.createClass({
             return <p>No upcoming events</p>;
         }
         return events.map(function(event) {
-            return <div key={event.uid} className="event-item">
-                <div className="event-summary">{event.summary}</div>
+            var loc = event.location ? <div className="event-location">{event.location}</div> : '';
```

Comment:
`null` is preferable to the the empty string here. ([Source](https://facebook.github.io/react/blog/2014/07/17/react-v0.11.html#rendering-to-null))